tests/tests_pytorch/loggers/test_mlflow.py

```diff
@@ -259,3 +259,23 @@ def test_mlflow_logger_experiment_calls(client, mlflow, time, tmpdir):
     logger._mlflow_client.create_experiment.assert_called_once_with(
         name="test", artifact_location="my_artifact_location"
     )
+
+
+@mock.patch("pytorch_lightning.loggers.mlflow.mlflow")
+@mock.patch("pytorch_lightning.loggers.mlflow.MlflowClient")
+def test_mlflow_logger_run_status_failed(client, mlflow):
+    class CustomModel(BoringModel):
+        def training_step(self, batch, batch_idx):
```

Comment:
You can raise the exception from an earlier hook like `on_fit_start`

Reply:
fixed the test to raise the exception on `on_fit_start`.